From 3cb29d4b4ecf950523adc6b603bf33444aec06fd Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:21 -0400
Subject: [redhat] kabi: Adding symbol pagevec_lookup (mm/swap.c)

Message-id: <b3cb8107d5931c20959bbf1846bb14e95e45714f.1410510773.git.poros@redhat.com>
Patchwork-id: 96099
O-Subject: [PATCH RHEL7.1 141/233] kabi: Adding symbol pagevec_lookup (mm/swap.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Lookup pages in the pagecache.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/pagevec_lookup
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/pagevec_lookup
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pagevec_lookup

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/pagevec_lookup b/redhat/kabi/kabi-module/kabi_ppc64/pagevec_lookup
new file mode 100644
index 0000000..58d92dc
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/pagevec_lookup
@@ -0,0 +1,2 @@
+#1-
+0x775aa293 pagevec_lookup vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/pagevec_lookup b/redhat/kabi/kabi-module/kabi_s390x/pagevec_lookup
new file mode 100644
index 0000000..49e4103
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/pagevec_lookup
@@ -0,0 +1,2 @@
+#1-
+0xd6e39a1e pagevec_lookup vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pagevec_lookup b/redhat/kabi/kabi-module/kabi_x86_64/pagevec_lookup
new file mode 100644
index 0000000..fd1d01e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pagevec_lookup
@@ -0,0 +1,2 @@
+#1-
+0x0229e570 pagevec_lookup vmlinux EXPORT_SYMBOL
-- 
1.7.1